Jenkinsfile

```diff
@@ -9,7 +9,7 @@ properties([[$class: 'BuildDiscarderProperty',
  *  https://github.com/jenkins-infra/documentation/blob/master/ci.adoc
  */
 Map branches = [:]
-['maven'/* TODO broken , 'maven-windows'*/].each {label ->
+['maven-11'/* TODO broken , 'maven-windows'*/].each {label ->
```

Comment:
but we will not build master with java 8... until we are jetty 9 and jenkins still java 8 we need to keep the build using java 8...

Reply:
Not true at all, Olivier. As long as `java.level` is set to 8, the Java 11 compiler will build the appropriate bytecode for existing use cases. Animal Sniffer enforces that we don't accidentally use newer APIs.

Reply:
I know but tests will be run using java 11.  and winstone have some weird code using some reflection on class from package `sun.security` . this is definitely working with java 11 but who knows...
I will definitely be happy to move to java 11 but until core still require java 8 I prefer we run the tests with java 8.
maybe a matrix?

Reply:
> maybe a matrix?

Running the `Jenkinsfile` build with both Java 8 and Java 11 makes sense to me. That is what most plugins and core do already. But how could I accommodate your request without write access to this repository? Any changes I make to the `Jenkinsfile` in this PR won't get executed because of my lack of write access. So I suggest either(a) you update the `Jenkinsfile` to test against both Java 8 and Java 11 (a PR which you, as the maintainer with write access, could test and iterate on), or (b) you give me write access so I can test the changes you have requested in this PR.

Reply:
> But how could I accommodate your request without write access to this repository

As a Jenkins core maintainer, you should have access to this repository.
If @olamy does not mind, I will update the permissions accordingly

Reply:
Thanks @oleg-nenashev, I'm still interested in getting the permissions updated if possible.

Reply:
Ping @olamy, / @jglick shall we give core write access?

Reply:
That's fine as long as this doesn't turn into crazy version bump which makes this non usable...

Reply:
I’ve updated the permissions for core

Reply:
> Not true at all, Olivier. As long as `java.level` is set to 8, the Java 11 compiler will build the appropriate bytecode for existing use cases. Animal Sniffer enforces that we don't accidentally use newer APIs.

Code should not rely on animal sniffer for JDK classes, rather use `-release` flag when on JDK9 or higher.  
There are also multi-release jars in any dependencies that will not be flagged using animal sniffer

Reply:
In the plugin pom this appears to be: https://github.com/jenkinsci/plugin-pom/blob/master/pom.xml#L959-L974

Not present in 'pom' though

Reply:
> That's fine as long as this doesn't turn into crazy version bump which makes this non usable...

Thank you for the vote of confidence, Olivier.

Reply:
> > That's fine as long as this doesn't turn into crazy version bump which makes this non usable...
> 
> Thank you for the vote of confidence, Olivier.

if you could give some time for people to review PRs that would be great.

Reply:
@timja @basil it's supposed to be a project with people working together? so it turns now to something to a one man show doing changes and not even waiting for approval on pull requets.
what a shame!!!

Reply:
Not sure why I’m tagged here…